medical_prescription_state_verify/models/medical_prescription_order.py

```diff
@@ -18,29 +18,16 @@ class MedicalPrescriptionOrder(models.Model):
             that has been defined in `_ALLOWED_CHANGE_STATES`)
         _ALLOWED_CHANGE_STATES: `list` of keys defining status types in which
             an RX can be moved to after being verified
+            4 - Verified
```

Comment:
Damn this strategy isn't going to work anymore without aggregating. Ok let's hold this & swoop back after `T0071 - Create base_kanban_stage_state`